util/cmp/compare.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Nit: year? Or maybe this is just copied from 2021 file?

Reply:
Lets update to 2022